BlueprintUI/Sources/Environment/AdaptedEnvironment.swift

```diff
@@ -0,0 +1,41 @@
+import CoreGraphics
+
+/// Wraps an element tree with a modified environment.
+///
+/// By specifying environmental values with this element, all child elements nested
+/// will automatically inherit those values automatically. Values can be changed
+/// anywhere in a sub-tree by inserting another `AdaptedEnvironment` element.
+public struct AdaptedEnvironment: Element {
```

Comment:
Should add an `extension Element` to match the SwiftUI-style stuff: (eg, https://github.com/square/Blueprint/blob/master/BlueprintUI/Sources/Layout/Aligned.swift#L118)